### DISCUSSION TITLE
[8일차 - 예진] "==" 와 "==="의 차이

### DISCUSSION THREAD

**Discussion by ez1n** [zzangCS/zzangInterview#28] [프론트 면접 질문]
"==" 와 "==="의 차이에 대해 설명해보세요.

**Comment by skyhanull** [comment]
'=='연산자는 값을 비교하고 '==='연산자는 값과 자료형까지 비교를 합니다
예시로는 '1'과 1이 있을 때 '=='는 값만 비교하므로 true가 되지만 '==='의 경우는 값은 같아도 자료형이 하나는 string 하나는 number라 서로 다르므로 false가 나옵니다

**Comment by ez1n** [comment]
**== 연산자**
값을 비교함

**=== 연산자**
값과 타입까지 함께 비교함
